cvat-ui/src/components/annotation-page/standard-workspace/styles.scss

```diff
@@ -91,6 +91,13 @@
     width: 100%;
 }
 
+.cvat-track-setting-popover-content {
+    padding: 10px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5609236)